### ISSUE TITLE
question on split pairs

### ISSUE THREAD

**Issue by caonetto** [nanoporetech/duplex-tools#43]
When running split pairs on pod5s and the basecalled sam, duplex tools generates a new folder with split pod5 files and associated read ids. Does this folder contain all the reads that where split, including non-duplex reads, or just the ones that where identified as duplex?

**Comment by caonetto** [comment]
Thank you for your quick response. Do you have a recommendation on how to split chimeric reads with midstrand adapters from dorado basecalled reads?

Thanks!

**Comment by ollenordesjo** [comment]
Yes, if you are ok with splitting reads in base-space (having input fastq & output fastq), then this tool should work for that: https://github.com/nanoporetech/duplex-tools/blob/master/duplex_tools/split_on_adapter.py. Feel free to give it a go and let me know if that is sufficient for the use case.
